repose-aggregator/core/repose-core/build.gradle

```diff
@@ -47,7 +47,15 @@ dependencies {
     compile "com.typesafe:config"
     compile "javax.ws.rs:javax.ws.rs-api"
 
-    provided "javax.servlet:javax.servlet-api"
+    compile "com.typesafe.play:play-json_$scalaMajDotMin"
+    compile "com.typesafe.akka:akka-actor_$scalaMajDotMin"
+    compile "org.eclipse.jetty:jetty-server"
+    compile "org.eclipse.jetty:jetty-servlet"
+    compile "net.sf.ehcache:ehcache-core"
+    compile "org.apache.abdera:abdera-core"
+    compile "org.apache.abdera:abdera-parser"
```

Comment:
Where did all these come from?

Reply:
They were of muddling through the transitives issue with the JSONPath stuff. They ended up not being necessary after fixing the real problem. They are now removed.